packages/@react-aria/tag/src/useTag.ts

```diff
@@ -69,7 +69,7 @@ export function useTag(props: TagProps<any>, state: GridState<any, any>): TagAri
   let domProps = filterDOMProps(props);
   return {
     clearButtonProps: mergeProps(pressProps, {
-      'aria-label': removeString,
+      'aria-label': `${removeString}: ${item.textValue || children}`,
```

Comment:
borrowed from v2, still a bit iffy on the messaging, kinda want to make it say "Double tap to remove" or something along those lines. Additionally, I noticed that you actually can't focus the clear button via VoiceOver/Talkback nor can you trigger removal by double tapping. Musing over two strategies:
1. Make it so that the clear button is focusable/navigable via VoiceOver/Talkback so user can double tap to remove that way
2. Make it so we detect if the click is dead center of the tag kinda like we do for Mobile Combobox

Reply:
We should structure this a little differently so that it translates well
```
      'aria-label': formatMessage('remove', {label: item.textValue || children}),
```
where the message 'remove' is defined as
```
{
  remove: 'remove: ${label}'
}
```

something like that, I forget the exact syntax

In addition, I'm not sure that a tag should read 'remove' anyways, I like your 'Double tap to remove' line of thinking. Like, here is the value you have in the tags section, it's interactable and that interaction will remove it from the tag list.

Reply:
`formatMessage` is called earlier in the hook to get a translated string for "remove", so do we need to do the above? I guess it would make it more clear if this becomes a "remove" announcement. cc @majornista do you have any opinions on the above two strategies?

Reply:
yeah, i shortened the calling of it, you can do that wherever as long as we only format the message once and it contains all aspects of the message, we're good. we just don't want to combine string ourselves because the order may not make sense in other languages